artemis-server/src/main/java/org/apache/activemq/artemis/core/server/impl/ServerSessionImpl.java

```diff
@@ -1259,6 +1259,7 @@ public RoutingStatus send(final ServerMessage message, final boolean direct, boo
       return send(getCurrentTransaction(), message, direct, noAutoCreateQueue);
    }
 
+   @Override
```

Comment:
@scop I just started my day but I guess you came up with the same changes :)

nice one!